DependencyInjection/Configuration.php

```diff
@@ -97,7 +97,7 @@ private function addDbalSection(ArrayNodeDefinition $node)
                             ->end()
                             ->children()
                                 ->scalarNode('class')->isRequired()->end()
-                                ->booleanNode('commented')->defaultTrue()->end()
+                                ->booleanNode('commented')->defaultFalse()->end()
```

Comment:
It's not really a BC break if people can still use the option.
Or should I wait for v2 for this change and v3 to remove the option entirely ?